app/models/grda_warehouse/tasks/identify_duplicates.rb

```diff
@@ -90,7 +90,7 @@ def identify_duplicates
           )
         end
         GrdaWarehouse::Hud::Client.import(
-          destination_client_updates,
+          destination_client_updates.index_by { |m| m[:id] }.values, # index to ensure no duplicate rows
```

Comment:
this is fine as is. Fwiw you can also use `uniq` with a block. I think that should be identical
```suggestion
          destination_client_updates.uniq { |m| m[:id] }, # ensure no duplicate rows
```

Reply:
Thank you!  I figured there was a more efficient way to unique them!